jira/client.py

```diff
@@ -92,7 +92,8 @@ def emit(self, record):
 #    "which means that there is a big change of having problems. " \
 #    "Possible workaround http://stackoverflow.com/a/17628350/99834" % encoding)
 
-logging.getLogger('jira').addHandler(NullHandler())
+logger = logging.getLogger('jira')
```

Comment:
`__name__` is `'jira.client'` here, not `'client'`.
If we're expecting people to reference the name of this logger then we need to be conscious that this logger name becomes part of our public API and a change to its name is a breaking change that will impact users of the library (silently), so perhaps having it explicitly hard-coded is not so bad?